new ffmpeg header structur
https://bugs.gentoo.org/show_bug.cgi?id=214740
Joerg Bornkessel <hd_brummy@gentoo.org> 29 Mar 2008
diff -Naur graphtft-0.1.16.alpha.orig/imlibrenderer/dvbrenderer/mpeg2encoder.c graphtft-0.1.16.alpha/imlibrenderer/dvbrenderer/mpeg2encoder.c
--- graphtft-0.1.16.alpha.orig/imlibrenderer/dvbrenderer/mpeg2encoder.c	2008-03-29 14:28:15.000000000 +0100
+++ graphtft-0.1.16.alpha/imlibrenderer/dvbrenderer/mpeg2encoder.c	2008-03-29 14:30:07.000000000 +0100
@@ -23,10 +23,10 @@
 
 extern "C" 
 {
-#include <avcodec.h>
+#include <libavcodec/avcodec.h>
 
 #ifdef HAVE_SWSCALE
-#  include <swscale.h>
+#  include <libswscale/swscale.h>
 #endif
 }
 
diff -Naur graphtft-0.1.16.alpha.orig/imlibrenderer/fbrenderer/fbrenderer.c graphtft-0.1.16.alpha/imlibrenderer/fbrenderer/fbrenderer.c
--- graphtft-0.1.16.alpha.orig/imlibrenderer/fbrenderer/fbrenderer.c	2008-03-29 14:28:15.000000000 +0100
+++ graphtft-0.1.16.alpha/imlibrenderer/fbrenderer/fbrenderer.c	2008-03-29 14:30:33.000000000 +0100
@@ -31,7 +31,7 @@
 #include <stdlib.h>
 
 #include <fbrenderer.h>
-#include <avcodec.h>
+#include <libavcodec/avcodec.h>
 #include <common.h>
 #include <setup.h>
 
diff -Naur graphtft-0.1.16.alpha.orig/imlibrenderer/fbrenderer/mpeg2decoder.c graphtft-0.1.16.alpha/imlibrenderer/fbrenderer/mpeg2decoder.c
--- graphtft-0.1.16.alpha.orig/imlibrenderer/fbrenderer/mpeg2decoder.c	2008-03-29 14:28:15.000000000 +0100
+++ graphtft-0.1.16.alpha/imlibrenderer/fbrenderer/mpeg2decoder.c	2008-03-29 14:31:00.000000000 +0100
@@ -23,10 +23,10 @@
 
 extern "C" 
 {
-#include <avcodec.h>
+#include <libavcodec/avcodec.h>
 
 # ifdef HAVE_SWSCALE
-#  include <swscale.h>
+#  include <libswscale/swscale.h>
 # endif
 }
 
diff -Naur graphtft-0.1.16.alpha.orig/imlibrenderer/fbrenderer/mpeg2decoder.h graphtft-0.1.16.alpha/imlibrenderer/fbrenderer/mpeg2decoder.h
--- graphtft-0.1.16.alpha.orig/imlibrenderer/fbrenderer/mpeg2decoder.h	2008-03-29 14:28:15.000000000 +0100
+++ graphtft-0.1.16.alpha/imlibrenderer/fbrenderer/mpeg2decoder.h	2008-03-29 14:31:14.000000000 +0100
@@ -19,7 +19,7 @@
 
 extern "C"
 {
-#include <avcodec.h>
+#include <libavcodec/avcodec.h>
 }
 #include <vdr/plugin.h>
 #include <vdr/ringbuffer.h>
diff -Naur graphtft-0.1.16.alpha.orig/Makefile graphtft-0.1.16.alpha/Makefile
--- graphtft-0.1.16.alpha.orig/Makefile	2008-03-29 14:28:15.000000000 +0100
+++ graphtft-0.1.16.alpha/Makefile	2008-03-29 14:28:32.000000000 +0100
@@ -11,7 +11,7 @@
 # You will need ffmpeg for dvb/fb-devices and for softmpeg, 
 # so install it and set path below:
 
-FFMDIR = /usr/include/ffmpeg
+FFMDIR = /usr/include
 
 #
 # Install imlib2 and uncomment the lines
